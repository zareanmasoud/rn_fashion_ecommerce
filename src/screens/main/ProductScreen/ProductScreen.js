import React, {Component} from 'react';
import {View} from 'react-native';
// import styles from './styles';

export class ProductScreen extends Component {
  render() {
    return <View />;
  }
}
