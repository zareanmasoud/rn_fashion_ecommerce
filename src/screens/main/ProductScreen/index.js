import ProductScreen from './ProductScreen';

export {ProductScreen as default};
