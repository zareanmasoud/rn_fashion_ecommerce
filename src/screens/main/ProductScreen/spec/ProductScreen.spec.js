import React from 'react';
import {shallow} from 'enzyme';
import ProductScreen from '../ProductScreen';

describe('ProductScreen', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<ProductScreen />);
      expect(component).toMatchSnapshot();
    });
  });
});
