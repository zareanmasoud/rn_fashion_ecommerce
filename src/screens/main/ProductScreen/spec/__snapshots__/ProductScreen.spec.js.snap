// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductScreen Rendering should match to snapshot 1`] = `
<ScreenContentFrame
  statusBackgroundColor="#ffffff"
  statusBarStyle="dark-content"
>
  <ProductHeader />
  <View
    flex={1}
  />
  <withNavigation(ProductTabbar) />
</ScreenContentFrame>
`;
